app/assets/stylesheets/default_theme/common.scss

```diff
@@ -0,0 +1,15 @@
+.well {
+  box-shadow: none;
+  border: 0;
+
+  > .form-group:last-child {
+    margin-bottom: 0;
+  }
+}
+
+// rubocop rule disabled so i can extend the existing Bootstrap class
+// scss-lint:disable PlaceholderInExtend
+.panel-header {
+  @extend .modal-header;
```

Comment:
Seems fair since we can't exactly modify the bootstrap code to use placeholders.  I'm surprised I didn't run into this with the datagrid styling yet.